### ISSUE TITLE
Add `ResNeXt [50, 101]` to keras.applications

### ISSUE THREAD

**Issue by innat** [keras-team/keras#15270]
**System information**.

TensorFlow version (you are using): 2.5
Are you willing to contribute it (Yes/No) : Yes

**Describe the feature and the current behavior/state**.

`ResNeXt` is a well-known classification model that is weirdly missing from the `keras.applications`. The idea that is presented [here](https://arxiv.org/abs/1611.05431) is great and I think it's fit for the model member in the keras package. 

**Will this change the current api? How?**
Yes. 

```
from tensorflow.keras.applications.resnext50 import ResNeXt50
from tensorflow.keras.applications.resnext101 import ResNeXt101
```

**Who will benefit from this feature?**
ML engineers and researcher who uses `tf.keras`. 

**Others**
Others implementation: https://github.com/qubvel/classification_models

**Comment by rchao** [comment]
Thanks for opening the issue. Can you provide a paragraph as a proposal for justifying adding this model? If you can provide some context of how widely this is used or some citation, that would be helpful.

**Comment by gcunhase** [comment]
Any update on this?

**Comment by innat** [comment]
[This model](https://arxiv.org/abs/1611.05431) was introduced in 2016 and now it's 2022 and still, we don't have it in `keras.application` even though it's cited by 6697.  Same as [its](https://github.com/keras-team/keras/issues/15267) tool.

**Comment by gcunhase** [comment]
How can that be?

**Comment by gcunhase** [comment]
@innat have you tried using ResNext from [here](https://github.com/qubvel/classification_models)? I'm not able to get good accuracies: https://github.com/qubvel/classification_models/issues/73 .

**Comment by innat** [comment]
I did use SE-ResNeXt most often. It worked well, comparable with DenseNet121.

**Comment by innat** [comment]
cc @taehoonlee 
Mentioning Lee who has once made a PR on ResNeXt HERE, https://github.com/keras-team/keras/pull/11203.

**Comment by gcunhase** [comment]
Thank you @innat !

**Comment by taehoonlee** [comment]
There are so many different types of deep learning architecture these days, (as I know) so François Chollet wants to add only those with special structural changes among them.

**Comment by innat** [comment]
@taehoonlee Is the decision still persist? The `torchvision` provides [these](https://pytorch.org/vision/stable/models.html) models. Models like wide-resnet, shuffle-net, unfortunately are missing as well.

**Comment by taehoonlee** [comment]
You can ask the maintainers. Now I am out of the keras team.
